sci-geosciences/grass/grass-8.0.0_rc1.ebuild

```diff
@@ -0,0 +1,271 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_REQ_USE="sqlite"  # bug 572440
+WX_GTK_VER="3.0-gtk3"
+
+inherit autotools desktop python-single-r1 toolchain-funcs wxwidgets xdg
+
+MY_PM=${PN}$(ver_cut 1-2 ${PV})
+MY_PM=${MY_PM/.}
+MY_P=${P/_rc/RC}
+
+DESCRIPTION="A free GIS with raster and vector functionality, as well as 3D vizualization"
+HOMEPAGE="https://grass.osgeo.org/"
+SRC_URI="https://grass.osgeo.org/${MY_PM}/source/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0/8.0"
+KEYWORDS="~amd64 ~ppc ~x86"
```

Comment:
I prefer to keep RCs unkeyworded usually so that people aren't subjected to beta software unless they want it, so I've made this one unkeyworded, but if you disagree, we can talk about it, of course. Maybe grass is OK with it and their RCs are more stable, but I dunno.

Reply:
It's OK. Thank you for the hint. I will bear it in mind in the future.

Reply:
I use/test dev version 8.1 (9999) and 8.0 RC1 every day, and it's fairly stable.

Reply:
@tmszi If as upstream, you think it's pretty reasonable for us to be shipping it in `~arch` (testing), then I'm OK with us doing that (but we won't mark RC versions stable).

Reply:
I agree with you, we will leave it without `~arch` (testing) for now.